lib/createTestFromScenario.js

```diff
@@ -85,6 +86,48 @@ const writeCucumberJsonFile = (json) => {
   cy.writeFile(outFile, json, { log: false });
 };
 
+const convertExamplesToScenarios = (scenario) => {
+  let counter = 0;
+  const hasEnvTags = !!getEnvTags();
+
+  return flattenArray(
+    scenario.examples.map((example) => {
+      const exampleTags = [...(example.tags || []), ...(scenario.tags || [])];
```

Comment:
This one is a critical change. The original code was: 

```javascript
example.tags.concat(section.tags)
```

But then my new tag test would report TypeErrors:

```none
 FAIL  lib/createTestFromScenario.test.js
  ● Console

    console.log
      Error parsing tags: '@test-tag'. Message: Cannot read property 'name' of undefined

      at shouldProceedCurrentStep (lib/tagsHelper.js:15:13)
          at Array.map (<anonymous>)
          at Array.map (<anonymous>)

    console.log
      Error parsing tags: '@test-tag'. Message: Cannot read property 'name' of undefined

      at shouldProceedCurrentStep (lib/tagsHelper.js:15:13)
          at Array.map (<anonymous>)
          at Array.map (<anonymous>)

  ● convertExamplesToScenarios › names scenarios uniquely

    TypeError: Cannot read property 'concat' of undefined

      93 |   return flattenArray(
      94 |     scenario.examples.map((example) => {
    > 95 |       const exampleTags = example.tags.concat(scenario.tags);
         |                                        ^
      96 | 
      97 |       return example.tableBody
      98 |         .map((row) => ({

      at map (lib/createTestFromScenario.js:95:40)
          at Array.map (<anonymous>)
      at convertExamplesToScenarios (lib/createTestFromScenario.js:94:23)
      at Object.<anonymous> (lib/createTestFromScenario.test.js:67:21)

  ● convertExamplesToScenarios › skips examples with non-matching tags

    TypeError: Cannot read property 'concat' of undefined

      93 |   return flattenArray(
      94 |     scenario.examples.map((example) => {
    > 95 |       const exampleTags = example.tags.concat(scenario.tags);
         |                                        ^
      96 | 
      97 |       return example.tableBody
      98 |         .map((row) => ({

      at map (lib/createTestFromScenario.js:95:40)
          at Array.map (<anonymous>)
      at convertExamplesToScenarios (lib/createTestFromScenario.js:94:23)
      at Object.<anonymous> (lib/createTestFromScenario.test.js:90:21)
```

I don't know how the other PR #557 missed this. @Sonaryr , FYI.

Reply:
Ah, sorry, I see what is wrong with my test. Wrong test data. I'll fix it.